### DISCUSSION TITLE
大佬求教，我想使用发送邮件的功能要怎么做？

### DISCUSSION THREAD

**Discussion by kledgold** [dreamhunter2333/cloudflare_temp_email#397] [General]
大佬求教，我想实现使用发送邮件的功能要怎么做？

现在会报这个错
Failed to send mail Mailchannels error: 500 {"errors":["Failed to send email: 550 5.7.1 This sender is not authorized to send from gaofast.email. See https://bit.ly/domain-lockdown. cfid=gaofast.email"]}

**Comment by dreamhunter2333** [comment]
建议参考文档中的 resend 部分，main channels 今天应该会停止服务

**Comment by dreamhunter2333** [comment]
参考此处的文档 https://temp-mail-docs.awsl.uk/zh/guide/config-send-mail.html
